app/controllers/report_controller.rb

```diff
@@ -8,6 +8,7 @@ class ReportController < ApplicationController
   include_concern 'Schedules'
   include_concern 'Widgets'
 
+  ReportController::DEFAULT_SORT_COL = 2
```

Comment:
```suggestion
  DEFAULT_SORT_COL = 2
```

Reply:
Also I'm thinking this should move above all of the include_concern lines (with a line break after it), because it is used within Reports and SavedReports